evm/gas-service.js

```diff
@@ -86,23 +100,26 @@ async function getGasUpdates(config, env, chain, destinationChains) {
             }
 
             const {
-                source_base_fee: sourceBaseFee,
-                source_express_fee: { total: sourceExpressFee },
-                source_token: {
-                    gas_price_in_units: { value: gasPrice },
+                destination_native_token: {
                     token_price: { usd: srcTokenPrice },
-                    decimals,
                 },
+            } = sourceFeeData;
+
+            const {
+                base_fee_usd: baseFeeUsd,
+                destination_express_fee: { total_usd: expressFeeUsd },
                 destination_native_token: {
-                    token_price: { usd: destinationTokenPrice },
+                    token_price: { usd: destTokenPrice },
+                    gas_price_in_units: { value: gasPriceInDestToken },
                 },
                 execute_gas_multiplier: multiplier = 1.1,
```

Comment:
```suggestion
                execute_gas_multiplier: executeGasMultiplier = 1.1,
```
to better distinguish it